flycheck-haskell.el

```diff
@@ -143,6 +143,9 @@ Return the configuration."
   (or (flycheck-haskell-get-cached-configuration cabal-file)
       (flycheck-haskell-read-and-cache-configuration cabal-file)))
 
+(defconst flycheck-haskell-cabal-config "cabal.config"
```

Comment:
> Ok, you're the expert

Not nearly, but thanks for the vote of confidence! TBH, I didn't know `cabal.config` existed either, until I needed the `with-compiler` setting.